demo/admin/src/products/future/CombinationFieldsTestProductsGrid.cometGen.ts

```diff
@@ -13,11 +13,7 @@ export const CombinationFieldsTestProductsGrid: GridConfig<GQLProduct> = {
             name: "titleAndCategory",
             headerName: "Title and Category",
             primaryText: "title",
-            secondaryText: {
-                type: "string",
-                field: "category.title",
-                emptyValue: "No category set",
-            },
```

Comment:
Is it okay to remove this advanced example?

Reply:
I only did this advanced example because the simple method didn't work yet 😅